package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "parent",
+  "version": "1.0.0",
+  "private": true,
+  "workspaces": [
+    "packages/*",
+    "examples/*"
+  ],
+  "scripts": {
+    "all": "yarn install && yarn lint && yarn test",
+    "build": "lerna run build",
+    "check:headers": "glsp checkHeaders . -t lastCommit",
+    "check:pr": "yarn all && yarn check:headers",
+    "clean": "lerna run clean && rimraf coverage .nyc_output",
+    "lint": "lerna run --parallel lint",
+    "lint:ci": "lerna run lint:ci",
+    "prepare": "lerna run prepare",
+    "publish:latest": "lerna publish from-git --no-git-reset --no-git-tag-version --no-verify-access --no-push",
+    "publish:next": "SHA=$(git rev-parse --short HEAD) && lerna publish preminor --exact --canary --preid next.${SHA} --dist-tag next --no-git-reset --no-git-tag-version --no-push --ignore-scripts --yes --no-verify-access",
+    "publish:prepare": "lerna version --ignore-scripts --yes --no-push",
+    "test": "lerna run test",
+    "test:ci": "lerna run test:ci",
+    "test:coverage": "lerna run test:coverage",
+    "test:coverage:ci": "yarn glsp coverageReport .",
+    "test:example": "cd examples/workflow-test && yarn test",
+    "upgrade:next": "yarn upgrade -p \"@eclipse-glsp.*|sprotty.*\" --next ",
+    "watch": "lerna run --parallel watch"
+  },
+  "devDependencies": {
+    "@eclipse-glsp/dev": "1.1.0-next.164cf99.124",
+    "@types/node": "16.x",
+    "eslint-import-resolver-typescript": "^3.5.3",
+    "eslint-plugin-import": "^2.27.5"
```

Comment:
I believe you should get that already through the dev package.

Reply:
yes, you are right